ui/js/services/main.js

```diff
@@ -128,17 +128,35 @@ treeherder.factory('thNotify', [
              * @sticky is a boolean indicating if you want the message to disappear
              * after a while or not
              */
-            send: function (message, severity, sticky, linkText, url) {
+            send: function (message, severity, opts) {
                 $log.debug("received message", message);
+                opts = opts || {};
                 severity = severity || 'info';
-                sticky = sticky || false;
+                var sticky = opts.sticky || false;
+
+                if (!_.isPlainObject(opts)) {
```

Comment:
This is a very kind way of correcting the developer.  :)  But, tbh, I'd rather we just throw an exception here if ``opts`` is passed in wrong.  Then the dev can say, "wth?" and look up the syntax and will know what to do.  :)

Obviously would need to do this check at the beginning of the function.